numpy/core/src/common/npy_cpu_features.c.src

```diff
@@ -40,11 +40,12 @@ NPY_VISIBILITY_HIDDEN int
 npy_cpu_init(void)
 {
     npy__cpu_init_features();
-    npy__cpu_validate_baseline();
-    npy__cpu_try_disable_env();
-
-    if (PyErr_Occurred())
```

Comment:
`PyErr_Occurred` function is called all over the repository, Is there any standard that It's should/shouldn't call. `a bunch of things` is kind of vague expression.

Reply:
> do a bunch of things then check 

I think the intent was "you should do one thing, then check `PyErr_Occurred` then do another thing, then check PyErr_Occurred`.

Reply:
Yes, @mattip's comment is correct - but conventionally you return -1 or NULL so that actually calling PyErr_Occurred isn't necessary.